python/lsst/obs/fiberspectrograph/rawFormatter.py

```diff
@@ -0,0 +1,43 @@
+__all__ = []
+
+from lsst.obs.base import FitsRawFormatterBase
+from .filters import FIBER_SPECTROGRAPH_FILTER_DEFINITIONS
+from ._instrument import FiberSpectrograph
+from .translator import FiberSpectrographTranslator
+import fitsio
+import astropy.units as u
+from lsst.daf.base import PropertyList
+
+
+class FiberSpectrographRawFormatter(FitsRawFormatterBase):
```

Comment:
(Probably history from when this obs package included the RubinGenericCamera)

Reply:
Sounds good - I am re-writing rawFormatter taking into account your other comments as well. 
If I don't inherit from `FitsRawFormatterBase`, where can I get `fileDescriptor` so I can get the file path?

Reply:
You can inherit from `Formatter` and implement the `read` method as in https://github.com/lsst/obs_base/blob/main/python/lsst/obs/base/formatters/fitsExposure.py#L90-L94

The `self.fileDescriptor` will be set for you. You can also implement `write` since you want `butler.put` to work and you can use the same formatter for both since I think they use the same StorageClass.